### PR TITLE
mce: Add custom lpm profile.

### PR DESCRIPTION
**Pull request by MagneFire**
Set the brightness to the minimum possible in ambient mode.
This appears to be what Android Wear also does to improve battery life.

### CODE REVIEW

File: recipes-nemomobile/mce/mce/20als-defaults.ini [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+LimitsProfile0=1;2;3;6;11;20;36;66;121;220;489;599;732;896;1095;1340;1639;2005;2453;3000
+LevelsProfile0=30;33;36;39;42;45;48;51;54;57;64;68;72;76;80;84;88;92;96;100
+
+[BrightnessLPM]
+
+# Use Limits and Levels profiles to select the desired brightness for a given ALS and current brightness.
+# When the device doesn't have an ambient light sensor the default value for 400 will be used(i.e. 417(closest) => 28%)
+# Profiles can be added to have different LPM brightness levels at different input brightness levels.
+# This sets the maximum brightness to 28% in ambient mode.
```
Comment (LittleFox94) [inline recipes-nemomobile/mce/mce/20als-defaults.ini]: I cannot link the comment to the values below - copy paste error?

Comment (MagneFire) [reply recipes-nemomobile/mce/mce/20als-defaults.ini]: This file is copied from the original: https://github.com/AsteroidOS/mce/pull/4/files#diff-705aa48c9445b9bdcc3d78edbff1b522R95

The values are just adjusted(improve battery life) to better suit smelt.

Comment (LittleFox94) [reply recipes-nemomobile/mce/mce/20als-defaults.ini]: Ah, so the comment does not add anything useful for smelt anymore?

Comment (MagneFire) [reply recipes-nemomobile/mce/mce/20als-defaults.ini]: Correct

Comment (LittleFox94) [reply recipes-nemomobile/mce/mce/20als-defaults.ini]: Thanks for clarifying :)

**Pull request merged**